ui/app/styles/components/modal.scss

```diff
@@ -87,6 +90,9 @@
 
 .modal-card-body {
   background-color: $white;
+  flex-grow: 1;
+  flex-shrink: 1;
```

Comment:
I'm a bit confused by these rules. What are we trying to accomplish here? 
Also we should make sure the modal content is all accessible (especially footers, since that's often where actions are) even if the browser is short

Reply:
This is for the modal that is extra long. I found this modal while looking at access->entities->add new policy->select ACL. Here is a screenshot of it on the branch with the css. 
![image](https://user-images.githubusercontent.com/6618863/231296904-58593c56-e269-4049-8373-78cc1359b95d.png)
(next I'll show on the binary to compare... not adding specific css, just copying exactly as it currently is).

Reply:
![image](https://user-images.githubusercontent.com/6618863/231297053-45da37f2-cafb-4881-ac51-81664a3a6570.png)

Reply:
For the most part, I am using the css that's on the binary. Hence the flex-grown and flex-shrink. The overflow was actually needed to make it perform like is intended. With this CSS the footers still seem accessible to me in a short browser.  Let me know if you think this still needs to be addressed (maybe a in a separate ticket?).

![image](https://user-images.githubusercontent.com/6618863/231507953-16833d92-3820-4078-ae0d-25818bb9f24f.png)

Reply:
Gotcha. Thanks for the extra context (and especially where you found that modal!) That makes sense to me 👍